vllm/engine/arg_utils.py

```diff
@@ -868,6 +869,16 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             help="Override or set the pooling method in the embedding model. "
             "e.g. {\"pooling_type\": \"mean\", \"normalize\": false}.'")
 
+        parser.add_argument('--compilation-config',
+                            '-O',
```

Comment:
I want to mimic the convention of traditional compiler, `-O 3` is the most optimized level.